src/SFA.DAS.AANHub.Application/Mediatr/Common/ValidationError.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SFA.DAS.AANHub.Application.Mediatr.Common
+{
+    public class ValidationError
```

Comment:
This has nothing to do with Mediatr, I think this is FE concern and hence should be inside API project

Reply:
Moved to API/Common